src/Broadway/EventStore/DBALEventStore.php

```diff
@@ -69,10 +72,41 @@ public function __construct(
     /**
      * {@inheritDoc}
      */
-    public function load($id)
+    public function load($id, $playhead = 0)
     {
         $statement = $this->prepareLoadStatement();
         $statement->bindValue('uuid', $this->convertIdentifierToStorageValue($id));
+        $statement->bindValue('playhead', $playhead);
+        $statement->execute();
+
+        $events = $this->executeStatement($statement);
+
+        if (empty($events)) {
+            throw new EventStreamNotFoundException(sprintf('EventStream not found for aggregate with id %s', $id));
+        }
```

Comment:
It would be good to test that aggregates can be loaded with only a snapshot and no new events.

The correct behaviour for `EventStore#load()` is to throw when there are truly no events for that ID in the event stream. In case of `$playhead !== 0`, this means it should actually check if there are no events at all for that ID. No events is not an exceptional case per se. Strictly, this would mean an extra query when no events are found and `$playhead !== 0`. I am not suggesting you implement this, I am merely outlining what is correct behaviour imo. Let's let the Qandidate guys take a look first.